frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -113,106 +112,23 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
                             {
                                 label: SurveyQuestionLabel[SurveyQuestionType.Open],
                                 value: SurveyQuestionType.Open,
-                                tooltip: () => (
-                                    <BaseAppearance
-                                        preview
-                                        onSubmit={() => undefined}
-                                        appearance={{
-                                            ...survey.appearance,
-                                            whiteLabel: true,
-                                        }}
-                                        question={{
-                                            type: SurveyQuestionType.Open,
-                                            question: 'Share your thoughts',
-                                            description: 'Optional form description',
-                                        }}
-                                    />
-                                ),
                             },
                             {
                                 label: 'Link/Notification',
                                 value: SurveyQuestionType.Link,
-                                tooltip: () => (
-                                    <BaseAppearance
-                                        preview
-                                        onSubmit={() => undefined}
-                                        appearance={{
-                                            ...survey.appearance,
-                                            whiteLabel: true,
-                                        }}
-                                        question={{
-                                            type: SurveyQuestionType.Link,
-                                            question: 'Do you want to join our upcoming webinar?',
-                                            buttonText: 'Register',
-                                            link: '',
-                                        }}
-                                    />
-                                ),
                             },
                             {
                                 label: 'Rating',
                                 value: SurveyQuestionType.Rating,
-                                tooltip: () => (
-                                    <SurveyRatingAppearance
```

Comment:
oh interesting, we're removing these, but would be nice to bring back in a future refactor. I'm assuming this was removed because the preview is now coming from the js package

Reply:
Issue created: https://github.com/PostHog/posthog/issues/21941